sheetfu/modules/table.py

```diff
@@ -206,3 +233,68 @@ def set_field_font_color(self, target_field, font_color_hex):
         if self.font_colors:
             self.font_colors[self.get_index(target_field)] = font_color_hex
         self.get_field_range(target_field).set_font_color(font_color_hex, batch_to=self.table)
+
+    def matches_value(self, header, value):
+        item_value = self.get_field_value(header)
+        return item_value == value
+
+
+class TableSelector:
```

Comment:
Super minor, I think it's worth to have table selector in a different file inside sheetfu/modules xD